docs/index.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Great Idea - Responsive I</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+  <header class="top-nav">
+    <nav>
+      <a class="nav-link" href="services.html">Services</a>
+      <a class="nav-link" href="">Product</a>
+      <a class="nav-link" href="">Vision</a>
```

Comment:
```suggestion
      <a class="nav-link" href="#">Vision</a>
```
Putting a # in your href on dead links will make it so that the page doesn't reload when you click on the link